packages/common/src/TransactionUtils.ts

```diff
@@ -1,6 +1,5 @@
 import util from "util";
 import minimist from "minimist";
-import ynatm from "@umaprotocol/ynatm";
```

Comment:
you can probs remove this from the associated `package.json` for `common`, no?

Reply:
it actually looks like `ynatm` is in a few other packages where it probs does not need to be. I dont see a point in it being in liquidator or trader. I think you can remove it from both of those places as well, no?

Reply:
done